website/src/scripts/header.js

```
@@ -0,0 +1,125 @@
+  drawerClose?.addEventListener('click', closeDrawer);
+  drawerScrim?.addEventListener('click', closeDrawer);
+
+  document.addEventListener('keydown', e => {
+    if (e.key === 'Escape' && document.body.classList.contains('drawer-open')) {
+      closeDrawer();
+    }
+  });
+
+  /* ── Branch toggle ──────────────────────────── */
+  const BRANCH_KEY = 'ag-branch';
+  const branchBtns = document.querySelectorAll('.branch-btn');
```

ESLint

[error] 'document' is not defined.

(no-undef)